test/views/wallet/ConfigureMobilecoindView/__snapshots__/ConfigureMobilecoindView.test.tsx.snap

```diff
@@ -90,7 +90,7 @@ exports[`Configure Mobilecoind View component render it renders correctly 1`] =
         <p
           class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextPrimary MuiTypography-displayInline"
         >
-          The ledger is synced.
+          The ledger is syncing. This may take awhile.
```

Comment:
This is fine. Our snapshots are based on the default test configurations -- and we're still locking in what that looks like. Later, we can explore if we want different snapshots based on different configurations. I would vote not-likely, because we explicitly unit tests those states. I see snapshots as a double check of unintentional changes to the DOM.